cmd/kubeadm/app/preflight/checks.go

```diff
@@ -346,3 +347,17 @@ func RunChecks(checks []PreFlightCheck, ww io.Writer) error {
 	}
 	return nil
 }
+
+func TryStartKubelet() error {
+	// If we notice that the kubelet service is inactive, try to start it
+	initSystem, err := initsystem.GetInitSystem()
+	if err != nil {
+		fmt.Println("[preflight] No supported init system detected, won't check if kubelet is running")
+	} else if !initSystem.ServiceIsActive("kubelet") {
+		fmt.Printf("[preflight] Starting the kubelet service by running %q\n", "systemctl start kubelet")
+		if err := initSystem.ServiceStart("kubelet"); err != nil {
+			return fmt.Errorf("Couldn't start the kubelet service. Please start the kubelet service manually and try again.")
+		}
```

Comment:
Will this error our if it's an systemd system, but kubelet is not managed by systemd? We should keep the door open for that possibility and let it work, so this should be a warning not an error. (I'm not sure if it will exit yet)